subprojects/core/src/main/java/org/gradle/api/internal/tasks/properties/ModifierAnnotationCategory.java

```diff
@@ -46,6 +47,9 @@
     ),
     IGNORE_EMPTY_DIRECTORIES("ignore empty directories",
         IgnoreEmptyDirectories.class
+    ),
+    IGNORE_LINE_ENDINGS("ignore line endings",
```

Comment:
This is not actually about ignoring line endings themselves, like we ignore empty directories, i.e. `alma\nalma` is not the same as `almaalma`. We already use "line ending normalization" in other places, I think we should standardize on that instead.

Reply:
I see your point.  I was just trying to maintain some consistency between the two, but there's not enough similarity there for it to make sense.  I've change it to `@NormalizeLineEndings`, `LineEndingSensitivity.NORMALIZE_LINE_ENDINGS` and `TaskInputFilePropertyBuilder.normalizeLineEndings()`.